app/views/users/show.html.haml

```diff
@@ -118,7 +118,7 @@
             %h4.account-page-card-title Projects Created
             - @user_projects.each do |project|
               .account-page-submitted-ontology
-                %a{href: "/ontologies/#{ont.acronym}"}= project.name
+                %a{href: "/ontologies/#{project.acronym}"}= project.name
```

Comment:
this link is not correct, it should be `/projects/#{project.acronym}`

Reply:
Ah okay thank you